src/main/webapp/WEB-INF/tags/instructor/studentList.tag

```diff
@@ -31,16 +31,21 @@
                     <th id="button_sortstudentname-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,4)">
                         Student Name <span class="icon-sort unsorted"></span>
                     </th>
-                    <c:if test="${not fromCourseDetailsPage}">
-                        <th id="button_sortemail-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,5)">
-                            Email <span class="icon-sort unsorted"></span>
-                        </th>
-                    </c:if>
-                    <c:if test="${fromCourseDetailsPage}">
-                        <th id="button_sortstudentstatus" class="button-sort-none" onclick="toggleSort(this,5)">
-                            Status <span class="icon-sort unsorted"></span>
-                        </th>
-                    </c:if>
+                    <c:choose>
+                        <c:when test="${fromCourseDetailsPage}">
+                            <th id="button_sortstudentstatus" class="button-sort-none" onclick="toggleSort(this,5)">
+                                Status <span class="icon-sort unsorted"></span>
+                            </th>
+                            <th id="button_sortemail-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,6)">
```

Comment:
@kennho could you look into opening your first issue - look into how to improve the ```toggleSort``` function to improve it - opening it doesn't mean you have to be the one to handle the issue